File: core/src/com/infeez/simple/PCInputProcessor.java
package com.infeez.simple;

public interface PCInputProcessor {

    void mouseMove(int x, int y);

    void mouseDown(int x, int y, int pointer, MouseButton button);

    void mouseUp(int x, int y, int pointer, MouseButton button);

    enum MouseButton {
        RIGHT,
        LEFT
    }
}

File: core/src/main/kotlin/com/infeez/simple/input/PCInputProcessor.kt
package com.infeez.simple.input

interface PCInputProcessor {
    fun mouseDrag(x: Float, y: Float, pointer: Int): Boolean

    fun mouseDown(x: Float, y: Float, pointer: Int, mouseButton: Int): Boolean

    fun mouseUp(x: Float, y: Float, pointer: Int, mouseButton: Int): Boolean

    fun touchCancelled(x: Float, y: Float, pointer: Int, mouseButton: Int): Boolean
}
